docs/Stream_Transformation_Functions.md

```diff
@@ -0,0 +1,31 @@
+# Stream transformation functions
+
+Create new stream transformation functions in Go. 
+
+## Required guidelines
+
+- Stream transformation functions must conform to the examples in [universe](../stdlib/universe) and include the required function and methods shown in the table below.
+- You must submit a unit test in in the same folder as the new stream transformation function implementation.
```

Comment:
Maybe a link to some tests: `stdlib/universe/columns_test.go`.

Each unit test should test if:

 - it is possible to create a new query from flux source with the new function that errors in the correct cases (`Test<functionname>_NewQuery`)
 - op spec marshaling (`Test<functionname>Operation_Marshaling`)
 - that the function actually works (`Test<functionname>_Process`)